### DISCUSSION TITLE
Custom taxonomy for attachments: no selection options in media modal screen.

### DISCUSSION THREAD

**Discussion by remcokalf** [pods-framework/pods#6722] [Ideas]
I have created a custom taxonomy for attachments with Pods, to create 'categories' for media items. 
All works well, the cats are showing up as a metabox with checkboxes in the media edit screen, as expected.

However, in the 'Attachment Details' modal screen (which opens when you click a media item from the gridview of the Media library, and after upload from a post), the categories do not show up as metabox or checkboxes or anything selectable. There is only a text input field, where you can add comma-separated categories. See the attached screenshot below.

Do you know of a way to get the checkboxes working here? 
Or could this be considered a feature request for Pods? Maybe this is WP core-related, or out of scope for Pods. I googled high and low but cannot find a solution. Some other media manager plugins get it working more or less, but add a lot of clutter or other issues.

![schermafbeelding 2016-09-04 om 16 07 31](https://cloud.githubusercontent.com/assets/4609920/18231422/bc85ab58-72b9-11e6-8a81-71a523f4115f.png)

**Comment by JoryHogeveen** [comment]
Not sure if WP supports metaboxes in the modal, don't think so.
I'll take a look at this.

**Comment by JoryHogeveen** [comment]
Allright, There are also more field types that are not working in the modal.
It apears that because in modal view everything is loaded through AJAX not the right .js files are included.
I'll make a new issue for this.

As for taxonomies, since WP doesn't support the default metaboxes for media modals either we should make a workaround for this or file a but-report at WordPress trac.
@sc0ttkclark @pglewis Any thoughts?

**Comment by JoryHogeveen** [comment]
Related: https://make.wordpress.org/core/2012/12/12/attachment-editing-now-with-full-post-edit-ui/

**Comment by JoryHogeveen** [comment]
Maybe also take a look at this for implementation:
https://nl.wordpress.org/plugins/enhanced-media-library/

**Comment by remcokalf** [comment]
Also related:

http://wordpress.stackexchange.com/questions/76720/how-to-use-taxonomies-on-attachments-with-the-new-media-library

This plugin also implements a solution with multiple select, but not checkboxes:
http://wordpress.org/plugins/attachment-taxonomies/

**Comment by JoryHogeveen** [comment]
Trac ticket since WP 3.5.. https://core.trac.wordpress.org/ticket/22938

**Comment by quasel** [comment]
Still in Progress? or did will/did the DFV stuff help there?
